app/src/androidTest/java/org/openobservatory/ooniprobe/ui/measurements/UIMeasurementsUtils.java

```diff
@@ -0,0 +1,54 @@
+package org.openobservatory.ooniprobe.ui.measurements;
+
+import org.openobservatory.ooniprobe.R;
+import org.openobservatory.ooniprobe.model.database.Measurement;
+import org.openobservatory.ooniprobe.model.database.Network;
+import org.openobservatory.ooniprobe.model.database.Result;
+import org.openobservatory.ooniprobe.utils.FormattingUtils;
+
+import static androidx.test.espresso.Espresso.onView;
+import static androidx.test.espresso.action.ViewActions.swipeLeft;
+import static androidx.test.espresso.assertion.ViewAssertions.matches;
+import static androidx.test.espresso.matcher.ViewMatchers.isDisplayed;
+import static androidx.test.espresso.matcher.ViewMatchers.withId;
+import static androidx.test.espresso.matcher.ViewMatchers.withText;
+import static org.hamcrest.Matchers.containsString;
+
+public class UIMeasurementsUtils {
```

Comment:
Not everything needs to be `Utils`. This could be `Assertions`. Or somethings this gets grouped by screens, like `MeasuramentScreen`.